new_changelog.txt

```diff
@@ -0,0 +1,2 @@
+Removed war option from Panay incident in mtg_usa.txt
```

Comment:
From my side, you don't need to add a separate changelog. I have a script that takes commit messages and compiles a changelog from them. If you don't want to include a given commit message in the changelog, just start it with "[NC]".

Reply:
noted